src/node.cc

```diff
@@ -1510,15 +1511,16 @@ void AppendExceptionLine(Environment* env,
 
   Local<String> arrow_str = String::NewFromUtf8(env->isolate(), arrow);
 
-  if (!arrow_str.IsEmpty() && !err_obj.IsEmpty() && err_obj->IsNativeError()) {
+  if (!arrow_str.IsEmpty() && !err_obj.IsEmpty() &&
+      (!handlingFatalError || err_obj->IsNativeError())) {
```

Comment:
This is hard to parse.  It'd be better to write it out like (for example) this:
```c++
if (!(handling_fatal_error || arrow_str.IsEmpty() || err_obj.IsEmpty() || !err->obj->IsNativeError())) {
```
Still not great but arguably more legible.